packages/components/addon/components/hds/page-header/index.hbs

```diff
@@ -0,0 +1,30 @@
+{{!
+Copyright (c) HashiCorp, Inc.
+SPDX-License-Identifier: MPL-2.0
+}}
+<div class="hds-page-header" ...attributes>
```

Comment:
is it possible to have the header be the outermost containing element?

We want to try to have all content contained in landmark elements; users with a screen reader can browse a page via landmarks, and a bookmark component outside of the header (one of the landmarks) is not ideal.

Reply:
The reason the header isn't the most outermost element is because it often wraps breadcrumbs which render a `nav` element. The intention was to not nest a landmark nav inside of a `header`.

Reply:
This was my recommendation, Melanie, to avoid nesting them, but if you think nesting the two landmarks is not a concern we can revert and apply the header element at the top level of this component.